io/js/src/main/scala/fs2/io/ioplatform.scala

```diff
@@ -305,7 +305,7 @@ private[fs2] trait ioplatform {
   /** Stream of bytes read asynchronously from standard input.
     * Takes a dummy `Int` parameter for source-compatibility with JVM.
     */
-  @nowarn("cat=unused")
+  @nowarn // ("cat=unused") - unsupported category on scala 3
```

Comment:
```suggestion
  @nowarn("msg=never used")
```
h/t @bplommer when we setup Scala 3 fatal warnings for http4s